data_model/src/lib.rs

```diff
@@ -426,6 +430,7 @@ impl Value {
             U32(_) | U128(_) | Id(_) | PublicKey(_) | Bool(_) | Parameter(_) | Identifiable(_)
             | String(_) | Name(_) | Fixed(_) | TransactionValue(_) | PermissionToken(_)
             | Hash(_) => 1_usize,
+            Block(v) => v.nested_len() + 1_usize,
```

Comment:
hm, where is +1 coming from?

Reply:
From counting the block itself, I guess?
As I said, I'm not sure I fully understand the semantics of this.